src/router/index.ts

```diff
@@ -1,6 +1,8 @@
 import { createRouter, createWebHistory } from "vue-router";
 import Splash from "../components/Splash.vue";
 import Signup from "../components/SignUp.vue";
+import PersonalInfo from "..//components/PersonalInfo.vue";
```

Comment:
Why the double slash in the file path? "..//components/PersonalInfo.vue". Curious to know